package-deploy.yaml

```diff
@@ -0,0 +1,129 @@
+tasks:
+  - task: docker.Version
+    configFilePath: scripts/configure_dev-deps.sh
+
+  - task: shell.docker.Ensure
+    name: deb
+    image: algorand/mule-linux-ubuntu
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/docker.ubuntu.Dockerfile
+    dependencies: docker.Version
+
+  - task: shell.docker.Ensure
+    name: rpm
+    image: algorand/mule-linux-centos
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/mule.go.centos.Dockerfile
+    dependencies: docker.Version
+
+  - task: docker.Make
+    name: generate-releases-page
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+    target: mule-deploy-generate_releases_page WORKDIR=/projects/go-algorand
+
+  - task: docker.Make
+    name: deb
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+      volumes: [
+        $HOME/packages:/root/packages,
```

Comment:
What is this packages folder for? We should only be writing the packages to the tmp folder inside of the project workspace so that we're not filling other directories on the host with files for this projects build process. The CI process will scale to more projects better that way

Reply:
Currently, the expectation is that the previous packages will be located in that folder.  The newest package is then copied to that directory, and then the snapshot is made.

This needs to be improved.  We'll need to determine from where we'll get all of those packages, and then this will be updated to that location.